src/poke_env/player/player.py

```diff
@@ -349,6 +354,9 @@ async def _handle_battle_message(self, split_messages: List[List[str]]):
                 await self._handle_battle_request(battle, from_teampreview_request=True)
             elif split_message[1] == "bigerror":
                 self.logger.warning("Received 'bigerror' message: %s", split_message)
+            elif split_message[1] == "uhtml":
+                if split_message[2] == "otsrequest":
```

Comment:
`uhtml` messages are currently handled in `battle.parse_message` - can you combine the two conditions (eg. `elif split_message[1] == "uhtml" and split_message[2] == "otsrequest"`) so that other messages are still parsed by it?